### PR TITLE
Automated releases

### LINKED ISSUE #2: Developer: Automated Releases
[Semantic Release](https://www.npmjs.com/package/semantic-release) provides a way for us to do automated releases. That's awesome, except it expects the Angular style of commit message. We use an Emoji commit style (https://github.com/punchcard-cms/punchcard/issues/1). We should be able to automatically generate releases using our commit style.

```gherkin
Feature: Automated Releases
  As a developer
  I want to be able to have the proper semantic release cut with changelog on successful merge
  So that I I do not have to cut releases on my own, ensuring better quality work more consistently.

  Scenario: Emoji Version Number
    Given that I write my commit messages with Emoji
     When I generate a new release
     Then it should know what emoji correspond to major/minor/patch releases

  Scenario: Emoji Changelog
    Given that I write my commit messages with Emoji
     When I generate a changelog for a new release
     Then it should know what emoji correspond to what sections of my changelog
```

---

- [ ] Automated Releases/Emoji Version Number
- [ ] Automated Releases/Emoji Changelog

### PR DESCRIPTION
**Pull request by Snugug**
Automated semantic releases

**Changelog**

```markdown
**New**

* Add Semantic Release Capabilities

**Changed**

* Update dependency versions
```

---
Resolves #70 
Resolves #2 
Resolves #65 
Resolves #59 

`DCO 1.1 Signed-off-by: Sam Richard <sam@snug.ug>`

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -27,11 +26,12 @@
...
     "dev:start": "gulp",
     "dev": "npm run dev:database & npm run dev:start",
-    "coverage": "nyc report --reporter=text-lcov | coveralls"
+    "coverage": "nyc report --reporter=text-lcov | coveralls",
+    "semantic-release": "semantic-release pre && npm publish && semantic-release post"
   },
   "repository": {
     "type": "git",
-    "url": "git@github.com:punchcard-cms/punchcard.git"
+    "url": "git+https://github.com/punchcard-cms/punchcard.git"
```
Comment (scottnath) [inline package.json]: Why is this here? Does one of the systems need to grab punchcard directly?

Comment (Snugug) [inline package.json]: Semantic Release does

Comment (Snugug) [inline package.json]: It's really changing it from the Git URL to the HTTPS URL

**Comment by Snugug** [comment]
Okay! This _should_ be working now!
